include/net/net_if.h

```diff
@@ -350,6 +350,12 @@ static inline enum net_verdict net_if_recv_data(struct net_if *iface,
 static inline u16_t net_if_get_ll_reserve(struct net_if *iface,
```

Comment:
This patch could be sent separately as this is not winc1500 related.

Reply:
It was part of original PR, plus: this driver is the one making use of that change.

In a way this PR is more an Net OFFLOAD show off than just a wifi driver only.

Reply:
If we are going to remove offload driver from l2, then we could do it right away. No need to wait for the winc1500 driver here. There are no other offloading device available so doing this removal is fine imho.

Reply:
But if you want to keep it part of this set, then that is fine too. It just looked like the first commit could be separated from this set.